go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/IBM/ibm-storage-odf-block-driver
 go 1.18
 
 require (
-	github.com/IBM/ibm-storage-odf-operator v1.3.0
+	github.com/IBM/ibm-storage-odf-operator v1.3.1-0.20230201081526-a62cc98b3818
```

Comment:
is that the tag we get for operator 1.4.0?

Reply:
yep, I'll try to update the tag after changing the operator.